src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -1868,4 +1879,4 @@ public function _js_vars() {
 
 		printf( "<script type='text/javascript'>list_args = %s;</script>\n", wp_json_encode( $args ) );
 	}
-}
+}
```

Comment:
Nit: Retain the line break at the end of the file.

```suggestion
}

```